style.css

```diff
@@ -174,11 +174,11 @@ input[type="submit"],
 textarea:focus,
 .button,
 .gallery img {
-	-webkit-transition: all 0.1s ease-in-out;
-	-moz-transition:    all 0.1s ease-in-out;
-	-ms-transition:     all 0.1s ease-in-out;
-	-o-transition:      all 0.1s ease-in-out;
-	transition:         all 0.1s ease-in-out;
+	-webkit-transition: all .1s ease-in-out;
```

Comment:
That's also fine, and I understand that the sample theme is a sort of allround starter theme, but it is a suggestion not a must.